### ISSUE TITLE
Cannot play audio in Windows

### ISSUE THREAD

**Issue by GokaiSanyu** [arkrow/PyMusicLooper#11]
```
E:\Music\Xenoblade Rips\Converted Files\Xenoblade 2>pymusiclooper "m41 - Gormott.ogg"
Playing with loop from 08:04.715 back to 03:39.552; similarity: 99.9%
(press Ctrl+C to stop looping.)
[src/libout123/module.c:INT123_open_module():185] error: Failure getting module directory! (Perhaps set MPG123_MODDIR?)
[src/libout123/libout123.c:out123_open():439] error: Found no driver out of [win32,win32_wasapi] working with device <default>.
ERROR: An issue related to the mpg123 library for playback has occured. If it is not installed/functional, alternatively use the export fucntionalities such as --export . See ` pymusiclooper --help ` for a full list of options.
```

How do I get the audio working with this? I have the MPG_123 libraries, but I don't know where I'm supposed to put them.

**Comment by arkrow** [comment]
Unfortunately, this is a known issue with mpg123 on windows (see issue #8). Fixing the issue requires a complete rewrite of the playback logic, as such an update for that is in the works but there is no ETA for it right now.

In the meantime, you can use the export function and use a music player with gapless playback support (e.g. foobar2000), or install pymusiclooper in the WSL environment (if you are already using it).

**Comment by arkrow** [comment]
Update: fixed in v3.0.0 thanks to a re-implementation of playback using a library with better cross-platform compatibility
